app.py

```diff
@@ -0,0 +1,1200 @@
+import random
+import time
+import pandas as pd
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import State, Input, Output
+import dash_daq as daq
+
+app = dash.Dash(__name__)
+
+# This is for Heroku
+server = app.server
+
+# CSS Imports
+external_css = ['./assets/style.css']
+
+for css in external_css:
+    app.css.append_css({"external_url": css})
+
+##############################################################################################################
+# Side panel
+##############################################################################################################
+
+satellite_dropdown = dcc.Dropdown(
+    id='satellite-dropdown-component',
+    options=[
+        {
+            'label': 'H45-K1',
+            'value': 'h45-k1'
+        },
+        {
+            'label': 'L12-5',
+            'value': 'l12-5'
+        },
+    ],
+    clearable=False,
+    # value='h45-k1',
+    style={
+        'color': '#017e84',
+        'text-align': 'center',
+        'cursor': 'pointer'
+    }
+)
+
+satellite_dropdown_text = html.P(
+    id='satellite-dropdown-text',
+    children=['Satellite Dashboard']
+)
+
+satellite_title = html.H1(
+    id='satellite-name',
+    children="")
+
+satellite_body = html.P(
+    className='satellite-description',
+    id='satellite-description',
+    children=[
+        ""
+    ]
+)
+
+side_panel_layout = html.Div(
+    id='panel-side',
+    children=[
+        satellite_dropdown_text,
+        html.Div(
+            id='satellite-dropdown',
+            children=[
```

Comment:
```suggestion
            children=satellite_dropdown
```
The `children` parameter can take single elements, too! (No need to wrap it in a list)